data/assets/scene/solarsystem/planets/earth/noaa-sos/land/forests.asset

```diff
@@ -0,0 +1,114 @@
+local globeIdentifier = asset.require("../../earth").Earth.Identifier
+
+
+local Name = "Forest Change (Extent, Gain, and Loss) - 2000 - 2014"
+local Identifier = "noaa-sos-land-forests"
+local Description = [[ This dataset shows annual tree cover extent,gain, and loss from 
+the year 2001 to 2014, at 30 meter resolution, as colored layers that can be seen 
+together or one at a time as individual layers that can be toggled on and off. Green is 
+used to represent tree cover in 2000, red shows tree cover loss between 2001-2014, 
+blue shows tree cover gain between 2001-2014, and purple is gain and loss together due to 
+replanting after loss has occurred.   ]]
+local URL = "https://sos.noaa.gov/catalog/datasets/forest-change-extent-gain-and-loss-2000-2014/"
+
+
+local syncedDirectory = asset.syncedResource({
+  Name = Name,
+  Type = "HttpSynchronization",
+  Identifier = Identifier,
+  Version = 1
+})
+
+local layer_combined = {
+  Identifier = Identifier,
+  Name = Name,
+  FilePath = syncedDirectory .. "combined.png",
+  Description = Description
+}
+
+local layer_extent = {
+  Identifier = Identifier .. "-extent",
+  Name = Name .. " (Extent)",
+  FilePath = syncedDirectory .. "extent.png",
+  Description = Description
+}
+
+local layer_gain = {
+  Identifier = Identifier .. "-gain",
+  Name = Name .. " (Gain)",
+  FilePath = syncedDirectory .. "gain.png",
+  Description = Description
+}
+
+local layer_loss = {
+  Identifier = Identifier .. "-loss",
+  Name = Name .. " (Loss)",
+  FilePath = syncedDirectory .. "loss.png",
+  Description = Description
+}
+
+local layer_map = {
+  Identifier = Identifier .. "-map",
+  Name = Name .. " (Map)",
+  FilePath = syncedDirectory .. "map.png",
+  Description = Description
+}
+
+local label = {
+  Identifier = Identifier,
+  Name = Name,
+  Type = "ScreenSpaceImageLocal",
+  TexturePath = syncedDirectory .. "forest_label.png",
+  FaceCamera = false,
+  Scale = 0.13,
+  CartesianPosition = { 0.5, 0.0, -2.0 }
+}
+
+local label1 = {
+  Identifier = Identifier .. "-1",
+  Name = Name .. "(1)",
```

Comment:
Renamed it anyway since I agree that it's not a good name:
```cpp
local label1 = {
  Identifier = Identifier .. "-alt",
  Name = Name .. " (alt)",
```